src/main/java/se/sundsvall/invoicesender/integration/raindance/RaindanceIntegration.java

```diff
@@ -31,22 +37,15 @@
 import org.apache.commons.io.IOUtils;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
-
-import se.sundsvall.invoicesender.model.Batch;
-import se.sundsvall.invoicesender.model.Item;
-
-import jcifs.CIFSContext;
-import jcifs.config.PropertyConfiguration;
-import jcifs.context.BaseContext;
-import jcifs.smb.NtlmPasswordAuthenticator;
-import jcifs.smb.SmbFile;
+import se.sundsvall.invoicesender.integration.db.entity.BatchEntity;
+import se.sundsvall.invoicesender.integration.db.entity.ItemEntity;
 
 public class RaindanceIntegration {
 
 	private static final Logger LOG = LoggerFactory.getLogger(RaindanceIntegration.class);
 
 	private static final DateTimeFormatter DATE_FORMATTER = DateTimeFormatter.ofPattern("yyMMdd");
-	private static final Predicate<Item> UNSENT_ITEMS = item -> item.getStatus() != SENT;
+	private static final Predicate<ItemEntity> UNSENT_ITEMS = item -> item.getStatus() != SENT;
 	private static final String BATCH_FILE_SUFFIX = ".zip.7z";
```

Comment:
You could use the `BATCH_FILE_SUFFIX` constant from the `Constants` class instead.